src/app/link-account/pages/institutional-linking/institutional-linking.component.html

```diff
@@ -0,0 +1,104 @@
+<main>
```

Comment:
We need to add 'id=main' here 

We have a related QA error https://trello.com/c/Oj2smo2Q/6777-qa-skip-to-main-content-doesnt-work-in-angular-pages
And a fix to add this in all other features https://github.com/ORCID/orcid-angular/pull/228